dashboard/client/src/pages/overview/OverviewPage.tsx

```diff
@@ -58,6 +69,44 @@ export const OverviewPage = () => {
       >
         <EventTable />
       </CollapsibleSection>
+
+      <CollapsibleSection
+        className={classes.section}
+        title="Autoscaler"
+        startExpanded
+      >
+        {
+          <div className={classes.overviewCardsContainer}>
+            <NodeCountCard className={classes.overviewCard} />
+            <OverviewCard
+              className={classNames(classes.root, classes.overviewCard)}
+            >
+              <NodeStatusCard cluster_status={cluster_status} />
+            </OverviewCard>
+            <OverviewCard
+              className={classNames(classes.root, classes.overviewCard)}
+            >
+              <ResourceStatusCard cluster_status={cluster_status} />
+            </OverviewCard>
+          </div>
+        }
+      </CollapsibleSection>
     </div>
   );
 };
+
+// <Grid container>
+// <Grid item xs={4}>
+//   <NodeCountCard className={classes.overviewCard} />
+// </Grid>
+// <Grid item xs={4}>
+//   <OverviewCard className={classes.root}>
+//     <NodeStatusCard cluster_status={cluster_status} />
+//   </OverviewCard>
+// </Grid>
+// <Grid item xs={4}>
+//   <OverviewCard className={classes.root}>
+//     <ResourceStatusCard cluster_status={cluster_status} />
+//   </OverviewCard>
+// </Grid>
+// </Grid>
```

Comment:
remove

Reply:
Sorry I was doing an experiment and forgot to remove the comment